service/src/main/java/io/camunda/service/JobServices.java

```diff
@@ -12,13 +12,20 @@
 import io.camunda.service.transformers.ServiceTransformers;
 import io.camunda.zeebe.broker.client.api.BrokerClient;
 import io.camunda.zeebe.gateway.impl.broker.request.BrokerActivateJobsRequest;
+import io.camunda.zeebe.gateway.impl.broker.request.BrokerFailJobRequest;
 import io.camunda.zeebe.gateway.impl.job.ActivateJobsHandler;
 import io.camunda.zeebe.gateway.impl.job.ResponseObserver;
+import io.camunda.zeebe.protocol.impl.record.value.job.JobRecord;
 import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.function.Consumer;
 
 public final class JobServices<T> extends ApiServices<JobServices<T>> {
 
+  public static final Integer DEFAULT_RETRIES_VALUE = 0;
+  public static final Long DEFAULT_RETRYBACKOFF_VALUE = -1L;
```

Comment:
Is a backoff of `-1` really appropriate for a user-facing API? A backoff of `0` I can understand as "no backoff, immediately retry". How do I interpret a backoff of `-1`? 🤔